doc/lpqcqp.rst

```diff
@@ -93,6 +93,9 @@ to indicate equality constraints.
     objective coefficient ``objcoef``. This is equivalent to calling the
     above method with empty arrays for the constraint coefficients.
 
+.. function:: delvars!(m::AbstractLinearQuadraticModel, idxs)
+
+    Removes the variables with indexes in the vector idxs.
```

Comment:
What happens to the remaining variables in the model? Should be explicit that they are renumbered.